### PR TITLE
Homework6

### PR DESCRIPTION
**Pull request by mikdikiy**

### CODE REVIEW

File: float/index.html [hunk 1]
```diff
@@ -1,59 +0,0 @@
-<!DOCTYPE html>
```
Comment (dmitry-darly-solutions) [inline float/index.html:1]: don't remove any files please

File: homework6/index.html [hunk 2]
```diff
@@ -0,0 +1,573 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Homework6</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="main-container">
+    <!--- HEADER -->
```
Comment (dmitry-darly-solutions) [inline homework6/index.html]: please remove such comments

File: homework6/style.css [hunk 3]
```diff
@@ -0,0 +1,510 @@
+/* homework6 */
+body{height: 100vh;width: 100vw;margin: 0;padding: 0;}
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: you don't need height: 100vh and width: 100vw on body

File: homework6/index.html [hunk 4]
```diff
@@ -0,0 +1,573 @@
...
+        <div class="basket">
+          <div class="basket-img"><img src="./img/basket.png"></div>
+          <span class="basket-text">товаров:</span>
+          <span class="basket-total-ware">0</span>
+          <span class="basket-text total-cash">на сумму:</span>
+          <span class="basket-total-cash">0,00</span>
+          <span class="basket-currency">грн.</span>
+        </div>
+      </nav>
+    </header>
```
Comment (dmitry-darly-solutions) [inline homework6/index.html:49]: image

File: homework6/index.html [hunk 5]
```diff
@@ -0,0 +1,573 @@
...
+  <title>Homework6</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="main-container">
+    <!--- HEADER -->
+    <header class="header">
+      <div class="header-panel">
+        <span class="header-telephone">
+          <img src="./img/telephone.png">(044)123-45-67 (044)567-89-10
```
Comment (dmitry-darly-solutions) [inline homework6/index.html]: wrap text in span

File: homework6/style.css [hunk 6]
```diff
@@ -0,0 +1,510 @@
...
+  margin-right: 10px;
+}
+.authorize-button, .authorize-text
+{
+  color:var(--authorize-button-color);
+  margin-right: 4px;
+  text-decoration: none;
+}
+.authorize-button:hover{text-decoration: underline;}
+.lost-parol{margin-left: 10px; color:var(--lost-parol-color)}
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid marging-left

File: homework6/style.css [hunk 7]
```diff
@@ -0,0 +1,510 @@
...
+  background: linear-gradient(-90deg, rgba(32,105,136,1) 0%, rgba(17,65,101,1) 59%, rgba(12,50,86,1) 100%);
+  border-radius: 10px;
+}
+.basket
+{
+  display: flex;
+  align-items:center; justify-content: flex-end;
+  font-family: Arial, sans-serif;
+  min-width: 216px;
+  margin-left: 70px; margin-bottom: 3px;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid margin-left

File: homework6/style.css [hunk 8]
```diff
@@ -0,0 +1,510 @@
...
+.by-tire-column{text-align: left;}
+.by-tire-check-text
+{
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', sans-serif;
+  font-size: 12px;
+}
+.tire-season-panel
+{
+  display:flex; flex-direction: row;
+  margin-left: 10px;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid marging-left

File: homework6/style.css [hunk 9]
```diff
@@ -0,0 +1,510 @@
...
+}
+.tire-season-container
+{
+  display: flex; flex-direction: column;
+  line-height: 1.1;
+  margin-left: 5px;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid marging-left

File: homework6/style.css [hunk 10]
```diff
@@ -0,0 +1,510 @@
...
+.by-tire-check
+{
+  vertical-align: middle;
+  transform: scale(0.8);
+  margin: 0;
+}
+.by-tire-header-string, .by-disc-header-string
+{
+  display: flex;
+  margin-left: 10px;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid marging-left

File: homework6/style.css [hunk 11]
```diff
@@ -0,0 +1,510 @@
...
+{
+  height: 16px;
+  display: flex; align-items: baseline;
+  padding-left: 0;
+  margin-bottom: 5px;
+}
+.catalog-list-text
+{
+  color:var(--catalog-list-text-color);
+  margin-left: 4px;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid marging-left

File: homework6/style.css [hunk 12]
```diff
@@ -0,0 +1,510 @@
...
+  align-items: center; justify-content: center;
+  width: 140px; height: 130px;
+  background-color: var(--wheel-img-back-color);
+  border: 1px solid #a4aeb1;
+  border-radius: 5%;
+  margin-left: auto; margin-right: auto;
+}
+.wheel-sale{text-align: center;}
+/* -----------BRAND----------- */
+.brand .aside-header{margin-left:15px;}
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid marging-left

File: homework6/style.css [hunk 13]
```diff
@@ -0,0 +1,510 @@
...
+.brand .aside-header:nth-child(1){margin-left:0;}
+.brand-list .catalog-check{margin-right: 15px;}
+.select-button
+{
+  margin-left: 27px;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid marging-left

File: homework6/style.css [hunk 14]
```diff
@@ -0,0 +1,510 @@
...
+
+.by-disc{width: 274px;}
+.by-disc-container{display: flex; flex-direction: row;}
+.by-disc-panel .label{width: 87px; text-align: left;}
+.by-disc-container select{width: 58px;}
+.by-disc-container .by-disc-hole {width: 52px;}
+.by-disc-panel{margin:0 10px 0 10px;padding: 0;}
+.by-disc-panel-plus
+{
+  margin-top: 2px;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid margin-top

File: homework6/style.css [hunk 15]
```diff
@@ -0,0 +1,510 @@
...
+  text-decoration: none;
+  margin-right: 15px;
+}
+.footer-button:hover{color: var(--footer-button-hover-color);}
+.site-creator
+{
+  font-size: 10px;
+  color: var(--site-creator-color);
+  position: absolute;
+  margin-top: 40px;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: avoid margin-top

File: homework6/style.css [hunk 16]
```diff
@@ -0,0 +1,510 @@
...
+  bottom: 5px;
+  right: 15px;
+}
+.find-ok:hover{border-bottom: 1px solid var(--find-ok-button-hover);}
+
+/* -----------FIND PANELS----------- */
+.by-mobile-content
+{
+  display: flex;
+  align-items: top; justify-content: space-between;
```
Comment (dmitry-darly-solutions) [inline homework6/style.css]: align-items: top; incorrect

File: homework6/index.html (hunk 4 above)
Comment (mikdikiy) [reply homework6/index.html:49]: In Chrome and Opera I can`t have this problem. Scroll appear when I use resize .

**Pull request closed without merging**